lib/backend/k8s/resources/ippool_test.go

```diff
@@ -134,14 +123,4 @@ var _ = Describe("IP Pool conversion methods", func() {
 		Expect(kvp.Value).To(BeAssignableToTypeOf(&model.IPPool{}))
 		Expect(kvp.Value).To(Equal(kvp1.Value))
 	})
-
```

Comment:
I'm not sure that's true.  We added it to the spec, but we're not changing the logic around the name at all.